docs/src/details/main-modules/images/sources.rst

```diff
@@ -0,0 +1,125 @@
+.. include:: /include/external_links.txt
+
+.. _image_sources:
+
+=============
+Image Sources
+=============
+
+The images module supports images in any of 3 forms:
+
+- **variables** (i.e. image data in ROM or RAM):  a :cpp:type:`lv_image_dsc_t` object
+  plus its data
+- **files** (strings beginning with a printable ASCII character are interpreted as a
+  file-system path to an image file)
+- **strings** containing Unicode characters
+
+.. container:: tighter-table-3
+
+    +-----------------------+--------------+------------+
+    | Form                  | Compressed?  | Location   |
+    +=======================+==============+============+
+    | variable (C struct)   | No           | ROM        |
+    +-----------------------+--------------+------------+
+    | variable (C struct)   | No           | RAM        |
```

Comment:
There are multiple built-in compressed image formats supported, e.g. (LZ4 or RLE)